metagpt/llm.py

```diff
@@ -8,9 +8,9 @@
 
 from metagpt.provider.anthropic_api import Claude2 as Claude
 from metagpt.provider.openai_api import OpenAIGPTAPI as LLM
+from metagpt.provider.spark_api import Spark
 
-DEFAULT_LLM = LLM()
-CLAUDE_LLM = Claude()
+DEFAULT_LLM = Spark()
```

Comment:
can't change the DEFAULT_LLM. What if the user want to use OPENAI_API. The better way is to setup a LLM which does't care about the API_CONFIG